libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -609,6 +640,24 @@ bool AP_Mount::get_attitude_euler(uint8_t instance, float& roll_deg, float& pitc
     return true;
 }
 
+bool AP_Mount::get_target_attitude_euler(uint8_t instance, float& roll_deg, float& pitch_deg, float& yaw_deg)
```

Comment:
Take a `Vector3f&` instead?

Reply:
I think we decided not to change this interface to use Vector3f for now.  Let's merge this method with the existing method used for scripting.